Makefile

```diff
@@ -133,7 +138,10 @@ update-ui:
 	@printf $(COLOR) "Install/update temporal ui-server..."
 	@go install github.com/temporalio/ui-server/cmd/server@latest
 
-update-tools: update-checkers update-mockgen update-proto-plugins
+update-tools: update-goimports update-linters update-mockgen update-proto-plugins update-proto-linters
+
+# update-linters is not included because in CI linters are run by github actions.
+ci-update-tools: update-goimports update-mockgen update-proto-plugins update-proto-linters
```

Comment:
This saves about 10s of build time because linters are not installed on Buildkite.